fastlane_bot/config/connect.py

```diff
@@ -18,6 +18,14 @@
 from dotenv import load_dotenv
 
 from fastlane_bot.config import network
+from fastlane_bot.config.constants import ARBITRUM_ONE_NETWORK, COINBASE_BASE_NETWORK, ETHEREUM_NETWORK, \
+    OPTIMISM_NETWORK, POLYGON_NETWORK, \
+    POLYGON_ZKEVM_NETWORK, \
+    WEB3_ALCHEMY_ARBITRUM, WEB3_ALCHEMY_BASE, \
+    WEB3_ALCHEMY_OPTIMISM, \
+    WEB3_ALCHEMY_POLYGON, \
+    WEB3_ALCHEMY_POLYGON_ZKEVM, \
+    WEB3_ALCHEMY_PROJECT_ID
```

Comment:
Semantic: I think that you can get rid of the backslashes by placing all imported symbols inside `()`.

Reply:
see revised